source/fuzz/fuzzer_util.h

```diff
@@ -60,8 +60,9 @@ bool PhiIdsOkForNewEdge(
 // to %some_block, with |bb_to| being the unreachable alternative.  Updates
 // OpPhi instructions in |bb_to| using |phi_ids| so that the new edge is valid.
 void AddUnreachableEdgeAndUpdateOpPhis(
-    opt::IRContext* context, opt::BasicBlock* bb_from, opt::BasicBlock* bb_to,
-    bool condition_value,
+    opt::IRContext* context,
+    const TransformationContext& transformation_context,
```

Comment:
Mention the new parameter in the comment for the function.

Reply:
Fixed in 8d9e990.